### DISCUSSION TITLE
How to start containers?

### DISCUSSION THREAD

**Discussion by Luxvao** [containerd/nerdctl#1079] [Q&A] [answered]
I can't seem to find a way to start a container. So if I want to run a container (and create a new one) I'd do "nerdctl run --name mainDebian -it debian". But if I want to run the existing container I can't find how to. So I make a container and then I have to write something to run it again in interactive mode.
Thanks in advance

**Comment by junnplus** [comment, marked as answer]
Try `nerdctl start <container>` and then execute exec to enter interactive mode.

**Reply by Luxvao** [reply]
If I remember correctly it doesn't support interactive or tty mode.

**Reply by junnplus** [reply]
> If I remember correctly it doesn't support interactive or tty mode.

Yes, but you can start the container and then execute exec to enter interactive mode.

**Reply by Luxvao** [reply]
Ok thanks
